avm/src/main.rs

```diff
@@ -33,13 +34,25 @@ fn main() {
       // .author(crate_authors!(", "))
       .arg_from_usage("<FILE> 'Specifies the file to load'"))
     .subcommand(SubCommand::with_name("compile")
-      .about("Compiles the given source file (.ln, .amm, .aga) to a new output file (.amm, .aga, .agc, .js)")
+      .about("Compiles the given source file (.ln, .amm, .aga) to a new output file (.amm, .aga, .agz, .agc, .js)")
       .arg_from_usage("<INPUT> 'Specifies the input file to load'")
       .arg_from_usage("<OUTPUT> 'Specifies the output file to generate'"))
     .subcommand(SubCommand::with_name("install")
       .about("Install '/dependencies' from '.dependencies.ln'")
       .version(crate_version!()))
-    .arg_from_usage("[SOURCE] 'Specifies a source ln file to compile and run'")
+    .subcommand(SubCommand::with_name("deploy-new")
```

Comment:
Yeah, I ended up just going with `alan deploy-new` in the spirit of the MVP because setting up nested subcommands for `alan deploy new` was a huge pain

Reply:
I mostly just prefer `alan deploy-new` over `alan deploy new` after actually using it, but I could use [this](https://docs.rs/clap/2.33.3/clap/struct.SubCommand.html#method.with_name) as @cdmistman pointed out

Reply:
So having them be separate names with this sort of grouping is kinda conventional across most CLI apps, and it's also nicer if/when we add autocomplete functionality for alan in bash and other shells so you can do `alan deploy<tab><tab>` and see the deploy commands instead of needing to do `alan<tab><tab>` and seeing *everything* including stuff you don't care about. Less of an issue now, but when we sprout package management commands on the alan command it'll start getting hard to follow.